src/View/Helper/FlashMessengerNamespace.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Mvc\Plugin\FlashMessenger\View\Helper;
+
+class FlashMessengerNamespace
```

Comment:
Should this be `@internal`?

Also, probably:

 * `final` 
 * `@psalm-immutable`

Reply:
If we mark this as `@internal`, we can mark all properties `public`, and remove most of the API of this object.

Reply:
@Ocramius , could you explain pls why we should mark all properties as `public`?
I don't see any benefit from this

Reply:
Since this class is `@internal`, and we only use it to convey state, if we mark it `@psalm-immutable`, all properties become read-only by default.

This means that there is no need for getters at all, and we reduce both runtime overhead and code size.

Note that `@psalm-immutable` is important here